samples/quickstart/requirements.txt

```
@@ -0,0 +1 @@
+google-cloud-datacatalog>=3.4.0
```

```suggestion
google-cloud-datacatalog==3.4.0
```

prefer pinning to versions and having renovate update

---

pinned these!